jobs/pacman-awsrules/src/main/java/com/tmobile/cloud/gcprules/vminstance/IPFowardingRule.java

```diff
@@ -90,42 +78,31 @@ private boolean checkIPForwardRule(String vmEsURL, Map<String, Object> mustFilte
         if (hitsJsonArray.size() > 0) {
             JsonObject vmInstanceObject = (JsonObject) ((JsonObject) hitsJsonArray.get(0))
                     .get(PacmanRuleConstants.SOURCE);
-
             logger.debug("Validating the data item: {}", vmInstanceObject.toString());
-
-            boolean canIPForward=true;
-            boolean isGKECluster=false;
-            if(vmInstanceObject.get(PacmanRuleConstants.CAN_IP_FORWARD) !=null){
-                 canIPForward=vmInstanceObject.get(PacmanRuleConstants.CAN_IP_FORWARD).getAsBoolean();
-                violationReason="IP Forward Enabled for  GCE  Instance";
-
-
-            }
-            else {
+            boolean canIPForward = true;
+            boolean isGKECluster = false;
+            if (!vmInstanceObject.get(PacmanRuleConstants.CAN_IP_FORWARD).isJsonNull()) {
+                canIPForward = vmInstanceObject.get(PacmanRuleConstants.CAN_IP_FORWARD).getAsBoolean();
+                violationReason = "IP Forward Enabled for  GCE  Instance";
```

Comment:
can you remove extra space around `GKE`?

Reply:
Addressed in commit